### PR TITLE
new list added

### PR DESCRIPTION
**Pull request by etech-staff**
please review

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1,13 +1,21 @@
 we are enjoying git
```
Comment (rnfor-pro) [inline file1:1]: correct

File: file1 [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+Broccoli
+Spinach
+Bell Pepper
+Cucumber
+==========
+DOMESTIC ANIMALS
 Dogs
 Cats
 Rabbits
-Guinea Pigs
```
Comment (rnfor-pro) [inline file1:12]: good one

File: file1 [hunk 3]
```diff
@@ -1,13 +1,21 @@
 we are enjoying git
 yet another line
+FRUITS
 Apple
 Banana
 Orange
 Mango
 Strawberry
 ==========
+VEGGIES
```
Comment (rnfor-pro) [inline file1:10]: I love this

**Review by rnfor-pro — APPROVED** [review]
wonderful job

**Comment by etech-staff** [comment]
ty!

**Pull request merged**